packages/framework-core/src/booster-command-dispatcher.ts

```diff
@@ -31,7 +34,14 @@ export class BoosterCommandDispatcher {
 
     const commandClass = commandMetadata.class
     this.logger.debug('Found the following command:', commandClass.name)
-    const commandInstance = createInstance(commandClass, commandEnvelope.value)
+
+    const commandInput = this.applyBeforeFunctions(
+      commandEnvelope.value,
+      commandMetadata.before,
+      commandEnvelope.currentUser
+    )
+
+    const commandInstance = createInstance(commandClass, commandInput)
     // TODO: Here we could call "command.validate()" so that the user can prevalidate
     // the command inputted by the user.
```

Comment:
I believe this comment is not necessary given we have hooks now, no?

Reply:
You're right!